adoc/reference-webui-systems.adoc

```diff
@@ -2935,22 +2943,26 @@ The following JSON data is an example of the exported information in the file:
 }
 ----
 
-
 For more information, see the man page on your {productname} server for [command]``virtual-host-gatherer``:
 
 ----
-{prompt.user}man virtual-host-gatherer
+man virtual-host-gatherer
 ----
 
 
-The `README` file coming with the package provides background information about the `type` of a hypervisor, etc.:
+The `README` file provided with the package provides background information about the `type` of a hypervisor, etc.:
 
 ----
 /usr/share/doc/packages/virtual-host-gatherer/README.md
 ----
 
-=== Configuring Virtual Host Managers via XMLRPC API
+The man page and the `README` file also contain example configuration files for your reference.
+
+
+
+=== Configuring Virtual Host Managers using the XMLRPC API
 
+In order to manage your VHMs with XMLRPC, you will need to configure them to use the XMLRPC APIs.
```

Comment:
VHMs do not really use the XMLRPC APIs, rather, VHMs can be configured either manually via the Web UI or programmatically via the XMLRPC APIs.